### PR TITLE
Add link out to an article's Talk page as an item on the footer menu

### PR DESCRIPTION
**Pull request by nambatee**
https://phabricator.wikimedia.org/T172747

Ready for review.

[page library PR](https://github.com/wikimedia/wikimedia-page-library/pull/90) has to be merged for this to work

### CODE REVIEW

File: Wikipedia/Code/WMFArticleViewController.m [hunk 1]
```diff
@@ -1502,6 +1505,15 @@ - (void)showEditHistory {
     [self presentViewControllerEmbeddedInNavigationController:editHistoryVC];
 }
 
+- (void)showTalkPage {
+    SFSafariViewController *safariController = [[SFSafariViewController alloc] initWithURL:self.articleTalkPageURL];
+    [self.navigationController presentViewController:safariController animated:YES completion:nil];
+}
+
+- (NSURL *)articleTalkPageURL {
+    return [NSURL URLWithString:[self.articleURL.absoluteString stringByReplacingOccurrencesOfString:@"/wiki/" withString:@"/wiki/Talk:"]];
```
Comment (montehurd) [inline Wikipedia/Code/WMFArticleViewController.m]: A little surprised `Talk:` work in other langs too - they redirect appropriately :)

Comment (nambatee) [reply Wikipedia/Code/WMFArticleViewController.m]: I know, I was pleasantly surprised too :D

Comment (montehurd) [reply Wikipedia/Code/WMFArticleViewController.m]: `replace` works here, but maybe consider using `wmf_URLWithTitle:` and `wmf_title` instead:

     [self.articleURL wmf_URLWithTitle:[@"Talk:" stringByAppendingString:self.articleURL.wmf_title]]

nice cause it accesses the components directly

**Comment by montehurd** [comment]
This looks good! Just the one wmf_title, comment but otherwise looks merge-able once the updated page lib bits are merged.

**Comment by montehurd** [comment]
@nambatee if you have a minute here's the page lib update:
https://github.com/wikimedia/wikipedia-ios/pull/1815

**Pull request merged**